### PR TITLE
No # in faq name

### LINKED ISSUE #2632: 4.0: No # in faq name
The symbol '#' should not be allowed in record titles as they are not working with htaccess-routes.

### PR DESCRIPTION
**Pull request by modelrailroader**
Fixes #2632

### CODE REVIEW

File: phpmyfaq/api.php [hunk 1]
```diff
@@ -359,6 +359,15 @@
             $categoryId = $categoryIdFound;
         }
 
+        if(str_contains($question, '#') >= 1) {
```
Comment (thorsten) [inline phpmyfaq/api.php:362]: You can use the method hasTitleAHash() from the class Faq here.

File: phpmyfaq/api.php [hunk 2]
```diff
@@ -359,6 +359,15 @@
...
+            $result = [
+                'stored' => false,
+                'error' => 'It is not allowed, that the question title contains a hash.'
+            ];
+            $response->setData($result);
```
Comment (thorsten) [inline phpmyfaq/api.php:367]: A response code is missing, 400 would be the best

**Review by thorsten — CHANGES_REQUESTED** [review]

**Comment by modelrailroader** [comment]
Changes done.

**Review by thorsten — APPROVED** [review]

**Comment by modelrailroader** [comment]
Have the spaces before ( in

**Pull request merged**